### ISSUE TITLE
Wording.

### ISSUE THREAD

**Issue by campbelllsssoup** [learn-co-curriculum/fewpjs-fewp-example#3]
In the second to last paragraph in this README, it states:
    
'This course will have your "web programmer" eyes will have you looking at your favorite sites very differently! '.

    I'd probably reword this to something like:
  
'This course will improve your "web programmer" vision and will have you looking at your favorite sites very differently! '. 

    It's a minor thing - but It helps me to learn and follow along when the literature is well written. Thank you for listening.

    -Matthew J. Campbell

**Comment by jenmyers** [comment]
Rewritten for clarity. Thank you for pointing it out!
